stdlib/Random/docs/src/index.md

```diff
@@ -18,9 +18,7 @@ The PRNGs (pseudorandom number generators) exported by the `Random` package are:
 Most functions related to random generation accept an optional `AbstractRNG` object as first argument.
 Some also accept dimension specifications `dims...` (which can also be given as a tuple) to generate
 arrays of random values.
-In a multi-threaded program, you should generally use different RNG objects from different threads
-or tasks in order to be thread-safe. However, the default RNG is thread-safe as of Julia 1.3
-(using a per-thread RNG up to version 1.6, and per-task thereafter).
+In a multi-threaded program, you should generally use different RNG objects from different threads or tasks in order to be thread-safe. However, the default global RNG is thread-safe as of Julia 1.3 (using a per-thread RNG up to version 1.6, and per-task thereafter). The global RNG refers to per-thread RNG instances; If you instantiate a non-global RNG, you should create a new RNG instance for each thread to avoid possible data-races.
```

Comment:
```suggestion
In a multi-threaded program, you should generally use different RNG objects from different threads or tasks in order to be thread-safe and deterministic. However, the default global RNG is thread-safe as of Julia 1.3 (using a per-thread RNG up to version 1.6, and per-task thereafter). The global RNG refers to per-thread RNG instances; If you instantiate a non-global RNG, you should create a new RNG instance for each thread to avoid possible data-races.
```
Thread-safety appears to be only about whether a process can crash. Often, determinism is also highly useful (e.g., https://github.com/JuliaAI/DecisionTree.jl/pull/174 were we copied the rng in each thread to ensure that runs with the same seed give the same outcome).

Reply:
What about:
"... to be thread-safe and deterministic (for a given seed)"?

Reply:
Isn't that sort of implied since the constructors of RNG objects are deterministic?

Reply:
Only if given a seed, though, right? 

```julia-repl
julia> using Random

julia> rand(Xoshiro())
0.801572038453949

julia> rand(Xoshiro())
0.8902731623152894
```

Reply:
Ah okay you’re right 👍